models/ht_neuron.cpp

```diff
@@ -618,21 +661,27 @@ ht_neuron::update( Time const& origin, const long_t from, const long_t to )
         S_.y_ );              // neuron state
 
       if ( status != GSL_SUCCESS )
+      {
         throw GSLSolverFailure( get_name(), status );
+      }
     }
 
     // Deactivate potassium current after spike time have expired
     if ( S_.r_potassium_ && --S_.r_potassium_ == 0 )
+    {
       S_.g_spike_ = false; // Deactivate potassium current.
+    }
 
     // Add new spikes to node state array
     for ( size_t i = 0; i < B_.spike_inputs_.size(); ++i )
+    {
       S_.y_[ 2 + 2 * i ] +=
         V_.cond_steps_[ i ] * B_.spike_inputs_[ i ].get_value( lag );
+    }
```

Comment:
@DimitriPlotnikov I see your point, but I like the loop because it is more flexible in case more synapse models would be added. I have added a more detailed comment to clarify things. I also removed the second line of the comment on line 577, since, after looking again at that code, I didn't really think it would make much sense to change function/variable names there.